### DISCUSSION TITLE
Unable to upload .ipynb to GitHub

### DISCUSSION THREAD

**Discussion by akshit0201** [community/community#7905] [Other Feature Feedback, Questions, & Ideas]
Whenever I upload .ipynb and click commit , the screen blanks out and the changes are not reflected. There is no problem when uploading the file  , only when committing.

**Comment by DocAndrew666** [comment]
![39E41FD0-2AF8-4326-8DBC-B87378B22952](https://user-images.githubusercontent.com/102442951/160261476-a70e7193-6ad5-47c9-851d-6d63d80861a7.png)
My changes still are unsaved.  I am unable to save what needs to be done.
